test/unit/mock/profiles/unsupported_dependencies/inspec.yml

```diff
@@ -0,0 +1,13 @@
+name: unsupported_dependencies
+title: InSpec Profile
+maintainer: The Authors
+copyright: The Authors
+copyright_email: you@example.com
+license: Apache-2.0
+summary: An InSpec Compliance Profile
+version: 0.1.0
+depends:
+  - name: child_profile
+    url: https://example.com/child_profile.tar.gz
```

Comment:
Consider using a `path` here instead, and pointing to the other fixture profile.